x-pack/plugins/security_solution/cypress/tasks/fields_browser.ts

```diff
@@ -34,17 +34,24 @@ export const addsHostGeoContinentNameToTimeline = () => {
 };
 
 export const clearFieldsBrowser = () => {
+  cy.clock();
   cy.get(FIELDS_BROWSER_FILTER_INPUT).type('{selectall}{backspace}');
+  cy.wait(0);
```

Comment:
Why is this wait doing? We should not use hardcoded waits. Moreover `cy.wait(0)` sounds like not doing a wait at all :)

Reply:
The reason is because the previous behavior of the fields browser attempted to render the rows of fields on every keystroke, so to fix this, the user input is now only applied 250 ms after the last keystroke, and only renders the rows once, at the end of typing, aka a debounce. It's not possible for cypress to have any a priori knowledge about what things the browser does are statically debounced, so we must use waits in situations like this https://stackoverflow.com/questions/69380424/cypress-test-with-debounce-input

Reply:
cy.wait(0) will allow cypress to release the js thread and allow any pending state/props/hooks updates to run, without it they are blocked, and things will remain in a loading state when they shouldn't, etc. Given the amount of business logic in hooks that are constantly running in security solution, a few cy.wait(0) would probably make multiple tests less flaky/faster imo